BraveWallet/Settings/WalletSettingsView.swift

```diff
@@ -60,8 +61,18 @@ public struct WalletSettingsView: View {
         }
       }
       .listRowBackground(Color(.secondaryBraveGroupedBackground))
+      Section(
+        footer: Text(Strings.Wallet.settingsResetTransactionFooter)
+          .foregroundColor(Color(.secondaryBraveLabel))
+      ) {
+        Button(action: { isShowingResetTransactionAlert = true }) {
+          Text(Strings.Wallet.settingsResetTransactionTitle)
+            .foregroundColor(Color(.braveLabel))
```

Comment:
This should be `brangeOrange` since its a button

Reply:
foregroundColor has been changed [8d8016d](https://github.com/brave/brave-ios/pull/5056/commits/8d8016d6f8d7241481bd86a5abab4598bebdea49)